dart/constraint/ConstraintSolver.cpp

```diff
@@ -31,6 +31,7 @@
  */
 
 #include "dart/constraint/ConstraintSolver.hpp"
+#include <algorithm>
```

Comment:
Nit: Please add an empty line before this line

Reply:
https://github.com/dartsim/dart/commit/5d81fbbc3a61d493f7f4f201657aa482ed169d2c